utility/src/main/java/org/oppia/util/logging/EventBundleCreator.kt

```diff
@@ -0,0 +1,66 @@
+package org.oppia.util.logging
```

Comment:
Please add tests for these functions!

Reply:
Relating this to my comment on the FirebaseEventLoggerTest, should I move those tests here and empty the wrapper tests ?

Reply:
This was discussed offline, resolving